Source/ARTRealtime+Private.h

```diff
@@ -68,6 +68,9 @@ NS_ASSUME_NONNULL_BEGIN
 
 @property (readonly, getter=getClientOptions) ARTClientOptions *options;
 
+/// Suspend the behavior defined in RTN15a, that is trying to immediately reconnect after a disconnection
+@property (readwrite, assign, nonatomic) BOOL suspendImmediateReconnection;
```

Comment:
Why this flag? If the client isn't respecting the spec then it should not be a flag to be activated only for the test suite, right? Maybe I'm missing something.

Reply:
@ricardopereira correct, now it waits 15s.

The flag is needed because sometimes you need a "disconnect and stay in the disconnected for x seconds" case. For example when you are testing that there's a new connection id if you connect after (ttl + idle interval). Without this flag you'd have an immediate connection retry.
We had to do [something similar in Java](https://github.com/ably/ably-java/commit/79dc37d6d2926a424311b96a715dcdf76e694f0e#diff-4cfebe2518c34c9f3975cde19f3ced08R469).

Reply:
Ok, understood.